src/nimble.nim

```diff
@@ -1043,8 +1043,10 @@ proc test(options: Options) =
   ## Subdirectories are not walked.
   var pkgInfo = getPkgInfo(getCurrentDir(), options)
 
+  let testsDir = getCurrentDir() / "tests"
+  var files: seq[typeof(walkDir(testsDir))]
```

Comment:
Please don't rely on the compiler to get the type of a variable.

Reply:
not sure why but fine, I've changed the code to avoid `typeof`.

Reply:
EDIT: you've closed my PR so github won't let me push my commit that fixes this